autogpt_platform/backend/backend/blocks/llm.py

```
@@ -8,7 +8,7 @@
 from abc import ABC
 from enum import Enum
 from json import JSONDecodeError
-from typing import Any, Iterable, List, Literal, Optional, cast
+from typing import Any, Iterable, List, Literal, Optional, cast, get_args
```

Ruff (0.16.3)

[warning] Import from `collections.abc` instead: `Iterable`

Import from `collections.abc`

(UP035)